cli/Cargo.toml

```diff
@@ -19,18 +19,18 @@ path = "src/bin/bendctl.rs"
 
 [dependencies]
 # Workspace dependencies
-databend-query = {path = "../query"}
-common-base = {path = "../common/base" }
+databend-query = { path = "../query" }
```

Comment:
Cool.
These pretty format is by tool or hand?

Reply:
by vscode plugin

Reply:
Emm, what's the plugin name? Thanks :D

Reply:
Even Better TOML